SPECS/libaec/libaec.spec

```
@@ -1,15 +1,14 @@
+Summary:        Adaptive Entropy Coding library
 Name:           libaec
 Version:        1.0.4
-Release:        4%{?dist}
-Summary:        Adaptive Entropy Coding library
-License:        BSD
+Release:        5%{?dist}
+License:        BSD 2-Clause
```

Thanks for switching to [SPDX identifiers](https://spdx.org/licenses/)! I think we only missed a hyphen here.
```suggestion
License:        BSD-2-Clause
```